packages/templates/react-urql-graphql-codegen/codegen.yml

```diff
@@ -0,0 +1,9 @@
+overwrite: true
+schema: "https://iteria-app-example01.herokuapp.com/v1/graphql"
```

Comment:
I am thinking how to replace also this endpoint during build time.

Reply:
hmm can we generate yml file before codegen?